server/fd/commands/__init__.py

```diff
@@ -9,6 +9,7 @@
 # at https://www.sourcefabric.org/superdesk/license
 
 from .resend_all import ResendAll  # noqa
+from .updatecl import Updatecl  # noqa
```

Comment:
please use firm camelCasing ;) and i would prefer speaking function names names like UpdateContentLists

Reply:
please read the https://www.python.org/dev/peps/pep-0008/ for naming conventions in python
i would look further into speaking function names